pf9/cluster/commands.py

```diff
@@ -172,7 +182,30 @@ def create(ctx, **kwargs):
 
     # Load active config
     Get(ctx).active_config()
-    Get(ctx).get_token_project()
+    Get(ctx).get_token_project_user_id()
+
+    segment_session = SegmentSession()
+    segment_event_properties = dict(arg_cluster_name=ctx.params['cluster_name'],
+                                    arg_master_ips=ctx.params['master_ip'],
+                                    arg_worker_ips=ctx.params['worker_ip'],
+                                    arg_user=ctx.params['user'], arg_password='REDACTED',
+                                    arg_ssh_key=ctx.params.get('ssh_key', None),
+                                    arg_masterVip=ctx.params.get('mastervip', None),
+                                    arg_masterVipIf=ctx.params.get('mastervipif', None),
+                                    arg_metallbIpRange=ctx.params.get('metallbiprange', None),
+                                    arg_containersCidr=ctx.params.get('containerscidr', None),
+                                    arg_servicesCidr=ctx.params.get('servicescidr', None),
+                                    arg_externalDnsName=ctx.params.get('externaldnsname', None),
+                                    arg_privileged=ctx.params.get('privileged', None),
+                                    arg_appCatalogEnabled=ctx.params.get('appcatalogenabled', None),
+                                    arg_allowWorkloadsOnMaster=ctx.params.get('allowworkloadsonmaster', None),
+                                    arg_networkPlugin=ctx.params.get('networkplugin', None),
+                                    arg_floating_ip=ctx.params.get('floating_ip', None),
+                                    keystone_user_id=ctx.params['user_id'],
+                                    du_account_url=ctx.params['du_url'])
+    SegmentSessionWrapper(ctx).load_segment_session(segment_session, segment_event_properties, "Create Cluster")
+    SegmentSessionWrapper(ctx).send_track("Load Active config")
+    segment_session.send_identify(ctx.params['du_username'], ctx.params['user_id'])
```

Comment:
Does username tracking lead to compliance issues? It will be personally identifiable information being tracked outside?

Reply:
Yea I spoke to charles and he suggested me to add it in EULA before we start pulling username, so I will remove for now, and see if we really need username.

Reply:
Removed